src/Microsoft.AspNet.Mvc.ViewFeatures/ViewFeatures/HtmlHelper.cs

```diff
@@ -272,12 +272,24 @@ public IHtmlContent AntiForgeryToken()
             FormMethod method,
             object htmlAttributes)
         {
+            // Push the new FormContext; GenerateEndForm() does the corresponding pop.
```

Comment:
Update these comments to include location of `GenerateEndForm()`. That method isn't in this class.